packages/mdl-button/index.scss

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "variables";
+
+.mdl-button {
```

Comment:
In general I think we can simplify these styles a lot. I [forked your pen](http://codepen.io/traviskaufman/pen/rLKppa?editors=1100) and applied some styles from a prototype I did a while back (which some stuff isn't perfect for either, like focus shades). It also removes the need for an inner content class.

The main things are: 

* For a button's active state, the interaction should actually be handled by the ripple. Therefore we don't need to apply any additional color treatments for that pseudo-class.
* For a raised button's active state, its elevation should transition, which is shown within the pen.
* Focus, unfortunately, is still one of those things that we still need more guidance on for web (see #4582), so I'm okay with holding off on performing any specific focus treatments for now. The way we got around this when prototyping was actually using javascript; we'd detect if a focus event was the result of a pointer event or not and if it was, we would hold off on applying a style.
* For disabled raised buttons, the background color doesn't change, only the text. Also, `pointer-events: none` or a disabling of all `:active` classes would have to be applied.
* I see that there are some button-element-specific resets that are missing e.g. `-webkit-appearance: none;` and `::-moz-focus-inner` treaments.